clio-client/src/test/resources/testdata/metadata

```diff
@@ -1,33 +1,33 @@
 {
-    "analysis_type" : "analysisType",
-    "bait_intervals" : "baitIntervals",
-    "data_type" : "dataType",
-    "individual_alias" : "individualAlias",
-    "initiative" : "initiative",
-    "lc_set" : "lcSet",
-    "library_type" : "libraryType",
-    "machine_name" : "machineName",
-    "molecular_barcode_name" : "mbName",
-    "molecular_barcode_sequence" : "mbSequence",
-    "paired_run" : true,
-    "product_family" : "productFamily",
-    "product_name" : "productName",
-    "product_order_id" : "productOrderId",
-    "product_part_number" : "productPartNumber",
-    "project" : "project",
-    "read_structure" : "readStructure",
-    "research_project_id" : "researchProjectId",
-    "research_project_name" : "researchProjectName",
-    "root_sample_id" : "rootSampleId",
-    "run_date" : "2017-08-09T14:21:57.995-04:00",
-    "run_name" : "runName",
-    "sample_alias" : "sampleAlias",
-    "sample_gender" : "sampleGender",
-    "sample_id" : "sampleId",
-    "sample_lsid" : "sampleLsid",
-    "sample_type" : "sampleType",
-    "target_intervals" : "targetIntervals",
-    "ubam_md5" : "ubamMd5",
-    "ubam_path" : "gs://testProject/testSample/ubamPath1.unmapped.bam",
-    "ubam_size" : 1
+  "analysis_type" : "analysisType",
+  "bait_intervals" : "analysisType",
```

Comment:
This might have been asked in an outdated comment, but were these all set to the same value on purpose?

Reply:
Nope. Copy/paste... I'll set them to more relevant values.